lib/new_relic/agent/new_relic_service.rb

```diff
@@ -346,6 +363,7 @@ def cert_file_path
           NewRelic::Agent.logger.warn("Couldn't find CA bundle from configured ca_bundle_path: #{path_override}") unless File.exist? path_override
           path_override
         else
+          ::NewRelic::Agent.increment_metric("Supportability/Ruby/Certificate/BundleRequired")
```

Comment:
Glad you moved this here; it will make sure it won't get recorded if the customer has provided a `ca_bundle_path`, and also, I think it's more intuitive to read.